src/main/java/domain/Deck.java

```diff
@@ -0,0 +1,34 @@
+package domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Deck {
+    private final Deque<Card> deck;
+
+    public Deck() {
+        deck = createShuffledDeck();
+    }
+
+    private List<Card> createCards() {
+        return Arrays.stream(Denomination.values())
+                .flatMap(denomination ->
+                        Arrays.stream(Suit.values())
+                                .map(symbol -> new Card(denomination, symbol)))
```

Comment:
```suggestion
                .map(this::createDenominationCards)
                .flatMap(List::stream)
```
스트림 내부의 동작도 너무 길어진다면 메서드로 분리해 볼까요?